Stanford/JumpstartEngineering/Install/Courses/DisableCourses.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @file
+ * Enable Courses
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Courses;
+
+/**
+ *
+ */
+class DisableCourses extends \ITasks\AbstractInstallTask {
+
+  /**
+   * Disable Courses module on JSE sites
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
```

Comment:
This task relies on the redirect and path modules. Please add a check for those at the top of the execute function and quit if the conditions are not met.